templates/ai/features.html

```diff
@@ -84,8 +84,10 @@ <h2>Tensorflow&trade;</h2>
       <p><a href="https://www.tensorflow.org/" class="p-link--external">Learn more about Tensorflow</a>
     </div>
     <div class="col-4 u-hide--small u-vertically-center">
-      <p><img src="{{ ASSET_SERVER_URL }}710ddd41-mnist_tensorboard.png" class="p-image--shadowed" alt="" /></p>
-      <p class="p-heading--six">Tensorflow from Google is officially published for Ubuntu</p>
+      <div>
+        <p><img src="{{ ASSET_SERVER_URL }}710ddd41-mnist_tensorboard.png" class="p-image--shadowed" alt="" /></p>
+        <cite>Tensorflow from Google is officially published for Ubuntu</cite>
```

Comment:
I think the semantic way of doing this would be `<figure>` and `<figcaption>`. An example (with a utility.... damn you Vanilla!)
``` html
<figure class="u-no-margin">
  <img src="{{ ASSET_SERVER_URL }}710ddd41-mnist_tensorboard.png" class="p-image--shadowed" alt=""> 
  <figcaption>Tensorflow from Google is officially published for Ubuntu</figcaption>
</figure>
```